packages/cycle-scripts/template/config/Typescript/flavor.js

```diff
@@ -0,0 +1,48 @@
+const dependencies = {
+  basics: [
+    '@cycle/dom@17.1.0'
+  ],
+  streamLib: {
+    xstream: [
+      '@cycle/run@3.1.0',
+      'xstream@10.5.0'
+    ],
+    rxjs: [
+      '@cycle/rxjs-run@7.0.0',
+      'rxjs@5.3.0'
+    ],
+    most: [
+      '@cycle/most-run@7.1.0',
+      'most@1.2.2'
+    ]
+  }
+}
+
+const replacements = {
```

Comment:
Wouldn't make sense to just move replacements within template as the dependencies cofnig is not related to this?